### DISCUSSION TITLE
How to enable rga-fzf

### DISCUSSION THREAD

**Discussion by Trid-collab** [phiresky/ripgrep-all#181] [Q&A]
I am new to rga. I using it Ubuntu 22.04 with the fish shell. Somehow running rga-fzf returns command not found, though rga works. I have also fzf installed and is working. Need help

**Comment by MC1R-13** [comment]
It's not built in. You have to create a script or a function.

[https://github.com/phiresky/ripgrep-all/wiki/fzf-Integration](url)
https://github.com/junegunn/fzf/blob/master/ADVANCED.md#ripgrep-integration
